---
#
# Use the widgets beneath and the content will be
# inserted automagically in the webpage. To make
# this work, you have to use › layout: frontpage
#
layout: frontpage

#
# Use the call for action to show a button on the frontpage
#
# To make internal links, just use a permalink like this
# url: /getting-started/
#
# To style the button in different colors, use no value
# to use the main color or success, alert or secondary.
# To change colors see sass/_01_settings_colors.scss
#

permalink: /index.html
#
# This is a nasty hack to make the navigation highlight
# this page as active in the topbar navigation
#
homepage: true
header:
  image: "VICC_color_combo_v2.png"
  background-color: "#18252B"
---

## The Variant Interpretation for Cancer Consortium
The field of precision medicine aspires to a future in which a cancer patient’s molecular information can be used to inform diagnosis, prognosis and treatment options most likely to benefit that individual patient. Many groups have created knowledgebases to annotate cancer genomic mutations associated with evidence of pathogenicity or relevant treatment options. However, clinicians and researchers are unable to fully utilize the accumulated knowledge derived from such efforts. Integration of the available knowledge is currently infeasible because each group (often redundantly) curates their own knowledgebase without adherence to any interoperability standards. Therefore, there is a clear need to standardize and coordinate clinical-genomics curation efforts, and create a public community resource able to query the aggregated information. To this end we have formed the *Variant Interpretation for Cancer Consortium (VICC)* to bring together the leading institutions that are independently developing comprehensive cancer variant interpretation databases. 

## Principles
VICC participants share a desire to coordinate efforts and thus enhance the value of each independent effort. Each participant has agreed to: (1) sharing at least a minimal set of required data elements for cancer variant interpretations; (2) Protecting patient privacy by focusing on only clinical interpretations of variants derived from published findings, not individual patient/variant-level observations, thus avoiding the possibility of linking variants to individuals; (3) Sharing all or a significant proportion of interpretations accumulated by their ongoing curation efforts; (4) Releasing content under a permissive license (free and non-exclusive for research use); (5) Releasing software in public repositories with open source licenses; (6) Making data available through publicly accessible and documented APIs and as cross-knowlegebase bulk downloads; and (7) Using, the existing schemas, APIs and demonstration implementations developed by GA4GH. 

## Leadership and Members
### Coming soon...

## Participating Organizations
To date a large number of academic and commercial groups have agreed to participate including those at Washington University (CIViC), MSKCC (OncoKB), Weill Cornell (PMKB), Oregon Health and Science University, Dana-Farber, Institute for Research in Biomedicine, Illumina Inc, and others. Together, this consortium will create a federated query service able to interrogate associations between cancer gene alterations and clinical actions, for each cancer disease setting, based on evidence amassed from all participating institutions world-wide.

## How to Join
### Coming soon...

## Ongoing Projects
### Coming soon...

