---
layout: page-fullwidth
title: "Projects"
meta_title: ""
subheadline: ""
teaser: ""
permalink: "/projects/"
header:
  image: "VICC_color_combo_v2.png"
  background-color: "#18252B"
---


Precision medicine promises a future in which a cancer patientâ€™s treatment is tailored to the specific molecular alterations of their tumor. Many groups are curating interpretations for which mutations are clinically relevant but their efforts are siloed, often redundant, and not interoperable. Therefore, there is a clear need to standardize and coordinate clinical-genomics curation efforts, and create a public community resource able to query the aggregated information. To address this challenge, we have formed the Variant Interpretation for Cancer Consortium (VICC) to promote global integration of knowledgebases for clinical interpretation of cancer variants. Together, this consortium will create a federated query service able to interrogate associations between cancer gene alterations and clinical actions, for each cancer disease setting, based on evidence amassed from all participating institutions world-wide. This will be accomplished through the following specific aims:

Aim 1. Harmonize global efforts for clinical interpretation of cancer variants and precision medicine clinical trial curation.  We will form an open consortium of developers and curators committed to eliminating the interpretation bottlenecks for precision medicine in cancer. 

Aim 1.1. Standardize data model for variant interpretation. We will establish the minimal data elements and standards needed to describe genotype, clinical phenotype, and evidence for clinically relevant genomic alterations in cancer.

Aim 1.2. Determine the minimal elements needed to define structured clinical trial eligibility criteria. We will develop a set of minimal elements needed for defining structured genomic and clinical eligibility criteria for cancer precision medicine clinical trials. We will also create algorithms to match patients to clinical trials based on genomic information.

Aim 1.3. Coordinate variant interpretation and clinical trial curation activities according to the domain expertise of different groups/institutes. We will attempt to unify efforts and leverage domain-specific expertise to reduce redundant curation effort. This will require developing methods for coordinating curation efforts, quality assurance, and clinician engagement.

Aim 2.  Implement software systems to query across standardized knowledgebases. Given a clinical sequencing assay result, for a patient with a specific cancer type, we will produce a comprehensive report of clinically relevant associations between genomic alterations and diagnosis, prognosis and treatment options using all publicly available sources of expert-curated interpretations.

Aim 2.1. Implement federated query for cross-knowledgebase interpretations. We will demonstrate ability to submit clinical genomics queries across disparate knowledgebases through a public interface.

Aim 2.2 Implement clinical trial query system. Building on the Clinical Trial Markup Language (CTML) and the NCI Clinical Trials API, we will build a public clinical trials portal that enables physicians and patients to search structured precision medicine clinical trials across multiple cancer centers. 

Aim 2.3 Implement web application for result exploration and report generation. We will develop capability to display clinically actionable recommendations based on cross-knowledgebase queries and accept contributions through a shared interface.

