---
layout: page-fullwidth
title: "Software"
meta_title: ""
subheadline: ""
teaser: ""
permalink: "/software/"
header:
    image_fullwidth: "genvis-dna-bg_optimized_v1a.png"
---

<div data-magellan-expedition="fixed">
  <ul class="sub-nav">
    <li data-magellan-arrival="CIViC"><a href="#CIViC">CIViC</a></li>
    <li data-magellan-arrival="pVACtools"><a href="#pVACtools">pVACtools</a></li>
    <li data-magellan-arrival="DGIdb"><a href="#DGIdb">DGIdb</a></li>
    <li data-magellan-arrival="GenVisR"><a href="#GenVisR">GenVisR</a></li>
    <li data-magellan-arrival="RegTools"><a href="#RegTools">RegTools</a></li>
    <li data-magellan-arrival="TrioMix"><a href="#TrioMix">TrioMix</a></li>
    <li data-magellan-arrival="VAtools"><a href="#VAtools">VAtools</a></li>
    <li data-magellan-arrival="OpenCAP"><a href="#OpenCAP">OpenCAP</a></li>
    <li data-magellan-arrival="DoCM"><a href="#DoCM">DoCM</a></li>
    <li data-magellan-arrival="CIViCpy"><a href="#CIViCpy">CIViCpy</a></li>

  </ul>
</div>

{% include software anchor="CIViC" title="CIViC: Clinical Interpretations of Variants in Cancer" logo="/assets/img/software/CIViC_logo.png" image="/assets/img/software/civic_screenshot.png" shorturl="civicdb.org" link="http://civicdb.org" description="CIViC provides a knowledgebase and curation system for expert crowdsourcing the clinical interpretation of variants in cancer." team="Malachi Griffith, Nicholas Spies, Kilannin Krysiak, Josh McMichael, Adam Coffman, Arpad Danos, Susanna Kiwala, Ben Ainscough, Erica Barnell, Alex Wagner, Zachary Skidmore, Robert Lesurf, Lee Trani, Avinash Ramu, Katie Campbell, Ajay Venigalla, Lana Sheta, Obi Griffith" source="civic-client (GitHub)" source_link="https://github.com/griffithlab/civic-client" source2="civic-server (GitHub)" source_link2="https://github.com/griffithlab/civic-server" citation="Griffith, Spies, Krysiak et al. 2017. CIViC is a community knowledgebase for expert crowdsourcing the clinical interpretation of variants in cancer. Nature Genetics. 49(2):170â€“174." citation_link="https://www.ncbi.nlm.nih.gov/pubmed/28138153" %}

{% include software anchor="pVACtools" title="pVACtools: Tools for Personalized Cancer Vaccine Design" logo="/assets/img/software/pvactools_logo.png" image="/assets/img/software/pVACtools_overview.png" shorturl="pvactools.org" link="http://pvactools.org" description="A computational toolkit for aiding design of personalized cancer vaccines." team="Susanna Kiwala, Huiming Xia, Megan Richters, Joshua McMichael, Christopher Miller, Jasreet Hundal, Yang-Yang Feng, Aaron Graubert, Alex Wollam, Amber Wollam, Connor Liu, Jason Walker, Obi Griffith, Malachi Griffith" source="pVACtools (GitHub)" source_link="https://github.com/griffithlab/pVACtools" citation="Hundal et al. 2016. pVAC-Seq: A genome-guided in silico approach to identifying tumor neoantigens. Genome Medicine. 8(1):11." citation_link="https://www.ncbi.nlm.nih.gov/pubmed/26825632" citation2="Hundal, Kiwala et al. 2020. pVACtools: A Computational Toolkit to Identify and Visualize Cancer Neoantigens. Cancer Immunology Research. 8(3):409-420." citation_link2="https://www.ncbi.nlm.nih.gov/pubmed/31907209" %}

{% include software anchor="DGIdb" title="DGIdb: The Drug-Gene Interaction Database" logo="/assets/img/software/dgidb_logo.png" image="/assets/img/software/dgidb_screenshot.png" shorturl="dgidb.org" link="http://dgidb.org" description="DGIdb offers user-friendly browsing, searching, and filtering of information on drug-gene interactions and the druggable genome, mined from over thirty trusted sources." team="Kelsy Cotto, Alex Wagner, Yang-Yang Feng, Susanna Kiwala, Adam Coffman, Gregory Spies, Alex Wollam, Nicholas Spies, Obi Griffith, Malachi Griffith" source="dgi-db (GitHub)" source_link="https://github.com/griffithlab/dgi-db" citation="Griffith, Griffith, et al. 2013. DGIdb: mining the druggable genome. Nature Methods. 10(12):1209-10." citation_link="https://www.ncbi.nlm.nih.gov/pubmed/24122041" citation2="Wagner et al. 2016. DGIdb 2.0: mining clinically relevant drug-gene interactions. Nucleic Acids Research. 44(D1):D1036-44." citation_link2="https://www.ncbi.nlm.nih.gov/pubmed/26531824" citation3="Cotto, Wagner et al. 2018. DGIdb 3.0: a redesign and expansion of the drug-gene interaction database. Nucleic Acids Research." citation_link3="https://www.ncbi.nlm.nih.gov/pubmed/29156001" citation4="Freshour, Kiwala, Cotto et al. 2021. Integration of the Drug-Gene Interaction Database (DGIdb 4.0) with open crowdsource efforts." citation_link4="https://pubmed.ncbi.nlm.nih.gov/33237278" %}

{% include software anchor="GenVisR" title="GenVisR: Genomic Visualizations in R" image="/assets/img/software/genvisr_overview.png"  shorturl="github.com/griffithlab/GenVisR" link="https://bioconductor.org/packages/release/bioc/html/GenVisR.html" description="GenVisR is a bioconductor package designed to allow users to produce highly customizable publication quality graphics for genomic data primarily at the cohort level." team="Zachary Skidmore, Alex Wagner, Katie Campbell, Kelsy Cotto, Jason Kunisaki, Obi Griffith, Malachi Griffith." source="GenVisR (Bioconductor)" source_link="https://bioconductor.org/packages/release/bioc/html/GenVisR.html" source2="GenVisR (GitHub)" source_link2="https://github.com/griffithlab/GenVisR" citation="Skidmore et al. 2016. GenVisR: Genomic Visualizations in R. Bioinformatics. 32(19):3012-4." citation_link="https://www.ncbi.nlm.nih.gov/pubmed/27288499" citation2="Skidmore et al. 2021. Exploring the Genomic Landscape of Cancer Patient Cohorts with GenVisR. Current Protocols." citation_link2="https://pubmed.ncbi.nlm.nih.gov/34506690" %}

{% include software anchor="RegTools" title="RegTools" image="/assets/img/software/regtools_overview.png" shorturl="regtools.org" link="http://regtools.org" description="RegTools is a set of tools that integrate DNA-seq and RNA-seq data to help interpret mutations in a regulatory and splicing context." team="Yang-Yang Feng, Avinash Ramu, Kelsy Cotto, Jason Kunisaki, Zachary Skidmore, Obi Griffith, Malachi Griffith" source="RegTools (GitHub)" source_link="https://github.com/griffithlab/regtools" %}

{% include software anchor="VAtools" title="VCF Annotation Tools" logo="/assets/img/software/vatools_logo2.png" image="/assets/img/software/vatools_screenshot.png" shorturl="vatools.org" link="http://vatools.org" description="VAtools is a python package that includes several tools to annotate VCF files with data from other tools." team="Susanna Kiwala" source="VAtools (GitHub)" source_link="https://github.com/griffithlab/VAtools" citation="In preparation" citation_link="https://github.com/griffithlab/VAtools" %}

{% include software anchor="TrioMix" title="TrioMix" image="/assets/img/software/triomix.png" shorturl="triomix.io" link="http://triomix.io/" description="TrioMix is a software tool for detecting intrafamilial DNA contamination, chimerism, and uniparental disomy from sequencing family trios." team="Chris Yoon, Obi Griffith, Malachi Griffith" source="TrioMix (GitHub)" source_link="https://github.com/cjyoon/triomix" citation="Yoon et al. 2022. Estimation of intrafamilial DNA contamination in family trio genome sequencing using deviation from Mendelian inheritance. Genome Research (2022)." citation_link="https://doi.org/10.1101/gr.276794.122" %}

{% include software anchor="OpenCAP" title="OpenCAP: Blueprints for Precision Medicine" logo="/assets/img/software/opencap_logo2.png" image="/assets/img/software/opencap_screenshot.png" shorturl="opencap.org" link="http://opencap.org" description="An open resource for building custom capture panels that are transparently linked to clinical variant annotations." team="Erica Barnell, Adam Waalkes, Kelsi Penewit, Katie Campbell, Zachary Skidmore, Colin Pritchard, Todd Fehniger, Ravindra Uppaluri, Ramaswamy Govindan, Malachi Griffith, Stephen Salipante, Obi Griffith" source="OpenCAP (GitHub)" source_link="https://github.com/griffithlab/civic-panel" citation="Barnell et al. 2019. Open-Sourced CIViC Annotation Pipeline to Identify and Annotate Clinically Relevant Variants Using Single-Molecule Molecular Inversion Probes. JCO Clin Cancer Inform. 3:1-12." citation_link="https://www.ncbi.nlm.nih.gov/pubmed/31618044" %}

{% include software anchor="DoCM" title="DoCM: Database of Curated Mutations" logo="/assets/img/software/docm_logo.png" image="/assets/img/software/docm_screenshot.png" shorturl="docm.info" link="http://docm.info" description="The Database of Curated Mutations (DoCM) is an open access resource that outlines literature supported somatic mutations that are clinically or biologically relevant." team="Ben Ainscough, Malachi Griffith, Adam Coffman, Alex Wagner, Jason Kunisaki, Joshua McMichael, Obi Griffith" source="docm (GitHub)" source_link="https://github.com/griffithlab/docm" citation="Ainscough et al. 2016. DoCM: a database of curated mutations in cancer. Nature Methods. 13(10):806-7." citation_link="https://www.ncbi.nlm.nih.gov/pubmed/27684579" %}

{% include software anchor="CIViCpy" title="CIViCpy: A software development kit (SDK) for the CIViC knowledgebase" logo="/assets/img/software/civicpy_logo.png" image="/assets/img/software/civicpy_workflow.png" shorturl="civicpy.org" link="http://civicpy.org" description="CIViCpy is designed to be an intuitive SDK and analysis toolkit for the Clinical Interpretations of Variants in Cancer (CIViC) knowledgebase." team="Alex Wagner, Susanna Kiwala, Adam Coffman, Joshua McMichael, Kelsy Cotto, Thomas Mooney, Erica Barnell, Kilannin Krysiak, Arpad Danos, Jason Walker, Obi Griffith, Malachi Griffith" source="CIViCpy (GitHub)" source_link="https://github.com/griffithlab/civicpy" citation="Wagner and Kiwala, et al. 2020. CIViCpy: A Python Software Development and Analysis Toolkit for the CIViC Knowledgebase. JCO Clin Cancer Inform. 4:245-253." citation_link="https://ascopubs.org/doi/pdf/10.1200/CCI.19.00127" %}
